util/update-version.sh

```diff
@@ -4,7 +4,7 @@ SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)
 PROJECT_DIR=$(dirname "$SCRIPT_DIR")
 
 # Fetch current version
-CURRENT_VERSION=$(sed -n '/^version\s*=\s*"\([^"]*\)"/{s//\1/p;q}' "$PROJECT_DIR/Cargo.toml")
+CURRENT_VERSION=$(sed -n '/^version\s*=\s*"\([0-9.]*\)"/{s//\1/p;q}' "$PROJECT_DIR/Cargo.toml")
```

Comment:
This change was to prevent 0.2.4-dev being interpreted as a newer version than 0.2.4.